api/OpenAI.net8.0.cs

```diff
@@ -5053,7 +5095,40 @@ public class ContainerFileCitationMessageAnnotation : ResponseMessageAnnotation,
         protected override BinaryData PersistableModelWriteCore(ModelReaderWriterOptions options);
     }
     [Experimental("OPENAI001")]
+    public class CreateResponseOptions : IJsonModel<CreateResponseOptions>, IPersistableModel<CreateResponseOptions> {
+        public CreateResponseOptions(List<ResponseItem> input);
+        public string EndUserId { get; set; }
+        public IList<Includable> Include { get; set; }
+        public IList<ResponseItem> Input { get; }
+        public string Instructions { get; set; }
+        public bool? IsBackgroundModeEnabled { get; set; }
+        public bool? IsParallelToolCallsEnabled { get; set; }
+        public bool? IsStoredOutputEnabled { get; set; }
+        public bool? IsStreamingEnabled { get; set; }
+        public int? MaxOutputTokenCount { get; set; }
+        public IDictionary<string, string> Metadata { get; }
+        public ModelIdsResponses? Model { get; set; }
```

Comment:
Historically, we don't use an enum for model names, and instead we keep it just as a string.